pkg/kubectl/resource/visitor.go

```diff
@@ -201,7 +204,7 @@ func ValidateSchema(data []byte, schema validation.Schema) error {
 		return fmt.Errorf("error converting to YAML: %v", err)
 	}
 	if err := schema.ValidateBytes(data); err != nil {
-		return fmt.Errorf("error validating data: %v", err)
+		return fmt.Errorf("error validating data: %v; %s", err, stopValidateMessage)
```

Comment:
How about an issue to create an `AggregatedValidationError` type so that we could plumb the handling for this all the way back to the caller?